tests/src/junit/java/net/neoforged/neoforge/unittest/transfer/UnsafeTransactionManagerTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.unittest.transfer;
+
+import net.neoforged.neoforge.transfer.EnergyHandlerUtil;
+import net.neoforged.neoforge.transfer.handlers.templates.energy.VoidEnergyHandler;
+import net.neoforged.neoforge.transfer.transaction.Transaction;
+import net.neoforged.neoforge.transfer.transaction.TransactionManager;
+import net.neoforged.neoforge.transfer.transaction.UnsafeTransactionManager;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class UnsafeTransactionManagerTest {
+    @Test
+    void unsafe() {
+        //Providing a way we can open transactions while inside a method that may not have the context available
+        try (Transaction transaction = UnsafeTransactionManager.openUnsafe()) {
+            Assertions.assertThat(transaction).isNotNull();
```

Comment:
@Matyrobbrt is assertj something that we use over junit's assertions in general in the tests? Because personally I feel like
```java
Assertions.assertNotNull(transactions);
Assertions.assertTrue(TransactionManager.isActive());
```
is a lot cleaner